StreamChat.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		465D64C2231808A2006BAE42 /* AttachmentTypeTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 465D64C1231808A2006BAE42 /* AttachmentTypeTests.swift */; };
+		798D413C23ACC89500F39EF8 /* ClientLoggerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 798D413B23ACC89400F39EF8 /* ClientLoggerTests.swift */; };
```

Comment:
@buh is it normal to have this in Git?

Reply:
I mean `project.pbxproj` not that HEX code specifically ;)

Reply:
Yes, it's ok. The test file should be in the test target.